docs/guide-end-of-form.rst

```diff
@@ -0,0 +1,243 @@
+* (If data is highly sensitive or devices are not trusted) Turn on :guilabel:`Delete after send`
+* (If data collection must be linear) :ref:`Disable moving backwards <moving-backwards-setting>`
+* (If data collectors may be tempted to change settings) Set an admin password and hide :guilabel:`User Settings` set above
+
+When data collectors reach the form end screen, they will only have the option to finalize. If they are interrupted during a form filling session, they will need to exit and discard changes or rely on automatic data backups and recovery (the partially-filled form will open automatically when they open the same blank form again).
+
+Edits are encouraged until a certain point in time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In some workflows, new information may need to be added to a form after a first data collection event. For example:
+
+* a form may capture multiple days' worth of interviews
+* a data collection subject such as a mountain may be observed from different angles at different times, revealing new information
+* review to fix small mistakes like typos may be encouraged
+* tasks like transcribing an audio recording may be needed
+
+To support this need, you can take the :guilabel:`Finalize` / :guilabel:`Send` button off of the form end screen and require that data collectors always use :ref:`bulk draft finalization <bulk-finalizing-drafts>`:
+
+* Disable "Finalize" from the end of form screen
+* (If data collectors may be tempted to change settings) Set an admin password
+* (If it's important to be able to block finalization of specific filled forms) Add a required yes/no question asking whether further edits are needed with a constraint that the answer must be ``No``
+* Train data collectors on using :ref:`bulk finalization <bulk-finalizing-drafts>`
+
+When data collectors reach the end of form screen, they only have the option of saving as draft. They can then make edits from the :guilabel:`Drafts` list as needed. When they are ready to submit, they go to :guilabel:`Drafts` and tap on the :guilabel:`Finalize all drafts` menu item. All forms marked with ``no errors`` will be finalized and sent. If there are certain submissions that they know are not yet ready, they can edit them to cause a validation error. This is most convenient to do with a yes/no question asking whether further edits are needed.
+
+Supervisor review required before submission
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Having a trusted supervisor do an in-field review of filled in forms can increase data collectors' attention to detail and help catch missing or incorrect data before it is submitted. You can decide when this review happens: it could be the data collector's responsibility to connect with their supervisor at some frequency or the supervisor could be the one who initiates review. This process can be written into the form or communicated separately such as during a training.
```

Comment:
Having a trusted supervisor do an in-field review of filled in forms before submission can increase data collectors' attention to detail and help catch missing or incorrect data